### PR TITLE
Data alerts

### PR DESCRIPTION
**Pull request by oscarrinro**

### CODE REVIEW

File: src/DataMessenger/DataMessenger.js [hunk 1]
```diff
@@ -485,14 +485,14 @@ export function DataMessenger(options = {}) {
...
-            <button class="autoql-vanilla-chata-btn primary"
+            <button class="autoql-vanilla-chata-btn autoql-vanilla-primary"
             style="padding: 5px 16px; margin: 10px 5px 2px;">
                 Create a New Notification
             </button>
         `);
 
         button.onclick = () => {
-            var modalView = new NotificationSettingsModal(obj.options);
+           /*  var modalView = new NotificationSettingsModal(obj.options);
```
Comment (alexzhou-chata) [inline src/DataMessenger/DataMessenger.js:495]: Could you please delete the comment or leave an explanation such as 
//TO DO --- (Example: why this comment must be kept here? Maybe you want to uncomment those for later use, some reason like that)

File: src/Notifications/Components/DataAlertCreationModal/DataAlertCreationModal.js [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+export function DataAlertCreationModal({ queryResponse, authentication }) {
+  const container = document.createElement('div');
+  const stepContentContainer = document.createElement('div');
+  const summaryFooter = new SummaryFooter({ queryText: queryResponse.data.text })
+
+  this.steps = [];
+  this.currentStepIndex = 0;
+
+  const confirmDialogProps = {
+    title: 'Are you sure you want to leave this page?',
```
Comment (alexzhou-chata) [inline src/Notifications/Components/DataAlertCreationModal/DataAlertCreationModal.js:22]: Could these text string be moved to the specific file and also get Language localisation added? (Spanish)

File: src/Notifications/Components/DataAlertCreationModal/SummaryFooter/SummaryFooter.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import './SummaryFooter.scss';
+
+export function SummaryFooter({ queryText }) {
+  const container = document.createElement('div');
+  const background = document.createElement('div');
+  const querySummary = document.createElement('div');
+  const summaryText = document.createElement('strong');
+
+  summaryText.textContent = 'Your query: ';
```
Comment (alexzhou-chata) [inline src/Notifications/Components/DataAlertCreationModal/SummaryFooter/SummaryFooter.js:9]: language localization

File: src/Notifications/DataAlerts/Components/DataAlertEditModal/DataAlertEditModal.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+        withFooter: true,
+        destroyOnClose: true,
+    },
+    () => {},
+    onDiscard
+  );
+  
+  const deleteDataAlertItemHandler = () => {
+    ChataConfirmDialog({
+      title: 'Are you sure you want to delete this Data Alert?',
```
Comment (alexzhou-chata) [inline src/Notifications/DataAlerts/Components/DataAlertEditModal/DataAlertEditModal.js:83]: language localization

File: src/Notifications/NotificationFeed/Components/MoreOptionsPopup/MoreOptionsPopup.js [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+    return response
+  }
+
+  this.handleTurnOffDataAlert = async() => {
+    popup.close();
+    const response = await this.updateDataAlertStatus(DATA_ALERT_STATUSES.INACTIVE);
+
+    if(response.status === 200) {
+      new AntdMessage('You will no longer receive notifications like this', 3000);
```
Comment (alexzhou-chata) [inline src/Notifications/NotificationFeed/Components/MoreOptionsPopup/MoreOptionsPopup.js:81]: language localization

**Comment by NikkiMeganMoore** [comment]
I am merging this FYI, but some comments above still need to be addressed

**Pull request merged**